gordo_components/builder/mlflow_utils.py

```diff
@@ -262,11 +262,20 @@ def get_batch_kwargs(metadata: dict) -> dict:
     model_keys = ["model-creation-date", "model-builder-version", "model-offset"]
     param_list += get_params(build_metadata["model"], model_keys)
 
+    # Parse cross-validation split metadata
+    try:
+        splits = build_metadata["model"]["cross-validation"]["splits"]
+        param_list += get_params(splits, splits.keys())
+    except KeyError:
+        logger.debug(
+            "Key 'metadata.model.cross-validation.splits' not found found in metadata."
```

Comment:
This error isn't correct is it, referencing `metadata.model.cross-validation` when it would have been under `build-metadata`